alpha/external-web/app/routes/declare-your-visit.js

```diff
@@ -0,0 +1,69 @@
+var router = require('../routes')
+var client = require('../services/eligibility-client')
+var logger = require('../services/bunyan-logger')
+
+// Require file upload library.
+var multer = require('multer')
+var upload = multer({ dest: 'visit-stamp-uploads/' })
+
+router.get('/declare-your-visit/:claimant_id', function (request, response, next) {
+  var id = request.params.claimant_id
+  client.get(id, function (error, claimant) {
+    if (!error) {
+      response.render('declare-your-visit', { 'claimant': claimant })
+      next()
+    } else {
+      response.status(500).render('error', { message: error.message, error: error })
+      next()
+    }
+  })
+})
+
+router.post('/declare-your-visit/:claimant_id', upload.single('stamp'), function (request, response, next) {
+  var id = request.params.claimant_id
```

Comment:
This function should be refactored as it is too large. 

client.update is called twice so that should be pulled out into another function. 
The next function can be called once after all of the redirect/renders have occurred it doesn't have to be after each one.